extra-tools/index.md

```diff
@@ -68,7 +73,7 @@ Note that this filter rewrites the `group` key from each item. For instance, an
 }
 {% endhighlight %}
 
-... would look like this once the `groupName` filter has been used:
+To use the `groupName` filter:
```

Comment:
The code example underneath is the result of the transformation after the filter is applied, and not an usage example.
So this is wrong.